lib/omise/OmiseCardList.php

```diff
@@ -28,7 +28,7 @@ public function __construct($cards, $customerID, $publickey = null, $secretkey =
      */
     public function retrieve($id)
     {
-        $result = parent::execute($this->getUrl($id), parent::REQUEST_GET, self::getResourceKey());
+        list($result, $headers) = parent::execute($this->getUrl($id), parent::REQUEST_GET, self::getResourceKey());
```

Comment:
I happened to read that in PHP spec, it recommends that 
> In general, it is recommended not to rely on the order in which list() assignments occur, as this is an implementation detail that may change again in the future.

What do you think?

Also It seems PHP 5 and PHP 7 are not treating `list()` output as the same order (In PHP 5, list() assigns the values starting with the right-most parameter. In PHP 7, list() starts with the left-most parameter.
As we are currently still having some users that are running Omise-PHP with a PHP5.x-installed server. Should we should consider of not using `list()`?

Reply:
Interesting, but I don't think this is of any concern to us here.

The way we have used list, the order of these assignments will make no difference. We're doing simple assignments to variables, then using those after. We couldn't care less if `$headers` is assigned before `$results` or vice versa.

The only situations in which I could see this might be an issue is if you are assigning to properties of an object that has setter methods that have some side effects. The setters would be called in a different order depending upon the PHP version, and it's possible the overall end result would be different.